src/lucky_record/field.cr

```diff
@@ -20,11 +20,19 @@ class LuckyRecord::Field(T)
     message_string = message.call(@name.to_s, @value.to_s)
     add_error(message_string)
   end
-  
+
   def errors
     @errors.uniq
   end
 
+  def value
+    if @value == ""
```

Comment:
If you use “@value.blank?” It will also catch strings that are all spaces. You’d also have to check that it is a String though so it doesn’t catch other objects that implement blank. So maybe something like: @value.is_a? && @value.blank?

Reply:
I didn't think of those cases, I'll update it.